source/clang/package.d

```diff
@@ -500,6 +500,28 @@ struct Cursor {
         return clang_getCursorDisplayName(cx).toString;
     }
 
+    /**  Get the raw declaration comment for this referent, if one exists. */
+    auto raw_comment() @safe pure nothrow const {
+        import std.typecons: Nullable;
+        auto cxrawcomment = clang_Cursor_getRawCommentText(cx);
+        if (cxrawcomment.data != null) {
+            return Nullable!string(cxrawcomment.toString());
+        } else {
+            return Nullable!string.init;
```

Comment:
Why not return an empty string?

Reply:
Because an empty comment is not the same as no comment.

Reply:
What's the difference in practice?

Reply:
This one has no comment:
```c
int x = 1
```

This one has an empty comment:
```c
/**/
int x = 1
```

There is a difference here, which can be important to the users of this API. That's why LLVM itself returns `null` instead of an empty string. We want to preserve the meanings here.